src/components/Shares.vue

```diff
@@ -28,7 +28,7 @@
 		<NcAvatar v-for="uid in uniqueShareUIDs"
 			:key="uid"
 			:user="uid"
-			:display-name="uidDisplaynameMap[uid]"
+			:title="uidDisplaynameMap[uid]"
```

Comment:
Same here, duplicate with title, don't replace the display-name

Reply:
corrected, thanks!